packages/theme-patternfly-org/template.ejs

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
+    <%= htmlWebpackPlugin.tags.headTags %>
+  </head>
+  <body>
+    <div id="root">
+      <%= prerendering %>
+    </div>
+    <script
```

Comment:
Usually it is recommended that js should appear before the body closing tag, and css in the head tag

Reply:
Right, but we don't want to block loading the entire site for Algogia CSS. Usually it takes the user longer than the 50Kb of Algogia CSS loads before they click the search box.